variables.tf

```diff
@@ -47,6 +47,11 @@ variable "tre_data_bucket_write_access" {
   type        = list(string)
 }
 
+variable "da_tre_sample_data_bucket_read_access" {
+  description = "Roles that have write access to tre-data-bucket"
```

Comment:
`tre-data-bucket` might be read as referring to a real bucket, but as far as I understand the actual bucket name is `da-sample-data`?

Reply:
oh and the comment too!